.babelrc

```diff
@@ -2,7 +2,15 @@
   "presets": ["@babel/preset-env"],
   "plugins": [
     ["module-resolver", {
-      "root": ["./"]
+      "root": ["./"],
+      "alias": {
+        "db": "./db",
+        "core": "./core",
+        "middlewares": "./middlewares",
+        "routes": "./routes",
+        "controllers": "./controllers",
+        "utils": "./utils"
```

Comment:
Do we need alias when we already set `root` dir?

Reply:
Okay, it's not neccessary. Removed